Nodejs/Tests/Core.UI/BasicProjectTests.cs

```diff
@@ -993,6 +993,20 @@ public class BasicProjectTests : SharedProjectTest {
                 actual = 0;
             }
             Assert.AreEqual(expected, actual, "count[" + key + "]");
+        }
+
+        /// <summary>
+        /// Remove the label denoting browser-side code, node, or both depending on the content type.
+        /// </summary>
+        private static string removeLabel(string itemName) {
+            string[] labels = new string[3] { " (browser)", " (node)", " (node, browser)" };
+            foreach (var label in labels) {
+                if (itemName.EndsWith(label)) {
+                    itemName = itemName.Substring(0, itemName.Length - label.Length);
+                    break; // safe to break as there is at most one label per name
+                }
+            }
+            return itemName;
```

Comment:
Instead of messing around with the string (which is going to break if we ever change the string or the option is turned off), it's better to just turn off the option before running the test. e.g. 
https://github.com/Microsoft/nodejstools/blob/8169f1e6fe7c83e3f4061631a44de79b3e80ec73/Nodejs/Tests/Core.UI/NodejsBasicProjectTests.cs#L272